src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const arrRobot = Object.entries(robot);
+  const arrValues = Object.values(robot);
+
+  if ([...new Set(arrValues)].length !== arrValues.length) {
```

Comment:
looks strange
better use for in circle
for (const key in robot) => check if robot[key] !== null else add to object with key robot[key] value key